src/assets/RustAsset.js

```diff
@@ -35,7 +40,7 @@ class RustAsset extends Asset {
   }
 
   async parse() {
-    // Install rust toolchain and target if needed
+    // Checks if rust is installed otherwise throw error
     await this.installRust();
```

Comment:
does this still need to be done again here since you moved it to `installParserDependencies`?

Reply:
I guess not, wanted to keep that for the if installed check but it throws so it won't be necessary, i'll remove it